lib/cocina/models/builders/title_builder.rb

```diff
@@ -8,42 +8,44 @@ module Builders
       # TitleBuilder selects the prefered title from the cocina object for solr indexing
       class TitleBuilder # rubocop:disable Metrics/ClassLength
         extend Deprecation
-        # @param [[Array<Cocina::Models::Title,Cocina::Models::DescriptiveValue>] titles the titles to consider
+        # @param [Array<Cocina::Models::Title,Cocina::Models::DescriptiveValue>] titles the titles to consider
+        # @param [Array<Cocina::Models::FolioCatalogLink>] catalog_links the folio catalog links to check for digital serials part labels
```

Comment:
Non-blocking question: whose job should be to protect against passing Symphony catalog links along in this argument? Or is that a far-fetched or impossible scenario?

Reply:
Line 19 is supposed to make sure we're only dealing with Folio links. It's not very likely there would be a partLabel in Symphony--something would really have had to go wrong--but at least we're checking here. Seems to make sense here rather than all the other possible places.